### PR TITLE
enrichedParties and assetsEnriched

### PR DESCRIPTION
**Pull request by guscost**
@pcowgill Would be good to verify our capitalization of pseudotransactions across the board here and while we update ou-api.

### CODE REVIEW

**Review by ponyjackal — APPROVED** [review]

File: src/contextualizers/test/transactions/goldDestination-0x469f8c9c.json [hunk 1]
```diff
@@ -298,7 +298,7 @@
     ]
   },
   "decoded": null,
-  "pseudoTransactions": [],
+  "pseudotransactions": [],
```
Comment (pcowgill) [inline src/contextualizers/test/transactions/goldDestination-0x469f8c9c.json:301]: We decided to revert this change, we'll split out capitalization changes into a separate PR

**Review by pcowgill — CHANGES_REQUESTED** [review]
https://github.com/Once-Upon/context/pull/373/files#r1610202968
